client/state/document-head/selectors.js

```diff
@@ -78,3 +80,25 @@ export function getFormattedTitle( state ) {
 
 	return title + 'WordPress.com';
 }
+
+/**
+ * Returns an array of document meta objects as set by the DocumentHead
+ * component or addDocumentHeadMeta action.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Object[]}       Array of meta objects
+ */
+export function getMeta( state ) {
```

Comment:
Hmm, in the context of global selectors, should they come to reality, would `getMeta` be sufficiently unique? Seems less so than `getTitle`, though a similar argument could be made there.